analytical_engine/core/context/java_context_base.h

```diff
@@ -203,22 +206,35 @@ class JavaContextBase : public grape::ContextBase {
       }
 
       {
-        VLOG(1) << "Creating app object: " << app_class_name_;
-        app_object_ =
-            LoadAndCreate(env, url_class_loader_object_, app_class_name_);
+        VLOG(10) << "Creating app object: " << app_class_name_;
```

Comment:
Why change VLOG(1) -> VLOG(10)?
we have a regulation about this, VLOG(1) used for detailed status, VLOG(10) are for very verbose logging, e.g., line printing during scan a file. 
Maybe preserve using VLOG(1) would be better.

Reply:
Okay.